java/tests/com/google/privacy/differentialprivacy/BoundedSumTest.java

```diff
@@ -679,4 +680,299 @@ private static void testForBias(
+                    (Double) invocation.getArguments()[3], null, (Double) invocation.getArguments()[5]));
+    sum =
+            BoundedSum.builder()
+                    .epsilon(0.1)
+                    .noise(noise)
+                    .delta(0.5)
+                    .maxPartitionsContributed(1)
+                    .lower(1)
+                    .upper(10)
+                    .build();
+    sum.addEntry(10);
+    sum.computeResult();
+
+    assertThat(sum.computeConfidenceInterval(0.5)).isEqualTo(ConfidenceInterval.create(3.068528194400547, 16.931471805599454));
+  }
+
+  @Test
+  public void computeConfidenceInterval_computeResultWasNotCalled_throwsException_forLong() {
+    // Mock the noise mechanism. Since noise is not Laplace, nor Gaussian, delta will be passed
+    // as a value instead of null, in order to pass the checks.
+    when(noise.computeConfidenceInterval(anyDouble(), anyInt(), anyDouble(), anyDouble(), anyDouble(), anyDouble()))
+            .thenAnswer(invocation -> new LaplaceNoise().computeConfidenceInterval(
+                    (Long) invocation.getArguments()[0], (Integer) invocation.getArguments()[1], (Long) invocation.getArguments()[2],
+                    (Double) invocation.getArguments()[3], null, (Double) invocation.getArguments()[5]));
+    BoundedSum.builder()
+            .epsilon(EPSILON)
+            .delta(DELTA)
+            .noise(noise)
+            .maxPartitionsContributed(1)
+            .lower(1)
+            .upper(5)
+            .build();
+    sum.addEntry(10);
+    IllegalStateException exception = assertThrows(IllegalStateException.class, () -> {
+      sum.computeConfidenceInterval(0.1554684);
+    });
+    assertThat(exception)
+            .hasMessageThat()
+            .startsWith("computeResult must be called before calling computeConfidenceInterval.");
+  }
+
+  @Test
+  public void computeConfidenceInterval_computeResultWasNotCalled_throwsException_forDouble() {
+    // Mock the noise mechanism. Since noise is not Laplace, nor Gaussian, delta will be passed
+    // as a value instead of null, in order to pass the checks.
+    when(noise.computeConfidenceInterval(anyDouble(), anyInt(), anyDouble(), anyDouble(), anyDouble(), anyDouble()))
+            .thenAnswer(invocation -> new LaplaceNoise().computeConfidenceInterval(
+                    (Double) invocation.getArguments()[0], (Integer) invocation.getArguments()[1], (Double) invocation.getArguments()[2],
+                    (Double) invocation.getArguments()[3], null, (Double) invocation.getArguments()[5]));
+    BoundedSum.builder()
+            .epsilon(EPSILON)
+            .delta(DELTA)
+            .noise(noise)
+            .maxPartitionsContributed(1)
+            .lower(1)
+            .upper(5)
+            .build();
+    IllegalStateException exception = assertThrows(IllegalStateException.class, () -> {
+      sum.computeConfidenceInterval(0.1554684);
```

Comment:
what is the reason to use such alpha? why not default one? the same for the next test